src/test/scala/castalia/StubConfigParserSpec.scala

```diff
@@ -0,0 +1,63 @@
+package castalia
+
+import java.io.FileNotFoundException
+
+import castalia.model.{CastaliaConfig, ResponseConfig}
+
+//import castalia.model.{StubConfig}
+import org.scalatest.{Matchers, WordSpec}
+import spray.json.{JsString, JsValue}
+
+/**
+  * Created by mihaelaoprea on 02/12/15.
+  */
+class StubConfigParserSpec extends WordSpec with Matchers {
+
+  "A StubConfigParser" when {
+    "json file \"jsonconfiguredstub.json\" exists on classpath" should {

```

Comment:
Nice trick could be: `"""json file "jsonconfiguredstub.json" exists on classpath"""`